common/saiserialize.cpp

```diff
@@ -100,12 +100,34 @@ sai_serialization_map_t sai_get_serialization_map()
     map[SAI_OBJECT_TYPE_VLAN_MEMBER][SAI_VLAN_MEMBER_ATTR_PORT_ID] = SAI_SERIALIZATION_TYPE_OBJECT_ID;
 
     map[SAI_OBJECT_TYPE_TRAP][SAI_HOSTIF_TRAP_ATTR_PACKET_ACTION] = SAI_SERIALIZATION_TYPE_INT32;
-    map[SAI_OBJECT_TYPE_TRAP][SAI_HOSTIF_TRAP_ATTR_TRAP_CHANNEL] = SAI_SERIALIZATION_TYPE_INT32;
     map[SAI_OBJECT_TYPE_TRAP][SAI_HOSTIF_TRAP_ATTR_TRAP_PRIORITY] = SAI_SERIALIZATION_TYPE_UINT32;
+    map[SAI_OBJECT_TYPE_TRAP][SAI_HOSTIF_TRAP_ATTR_TRAP_CHANNEL] = SAI_SERIALIZATION_TYPE_INT32;
+    map[SAI_OBJECT_TYPE_TRAP][SAI_HOSTIF_TRAP_ATTR_FD] = SAI_SERIALIZATION_TYPE_OBJECT_ID;
+    map[SAI_OBJECT_TYPE_TRAP][SAI_HOSTIF_TRAP_ATTR_PORT_LIST] = SAI_SERIALIZATION_TYPE_OBJECT_LIST;
+    map[SAI_OBJECT_TYPE_TRAP][SAI_HOSTIF_TRAP_ATTR_TRAP_GROUP] = SAI_SERIALIZATION_TYPE_OBJECT_ID;
 
     map[SAI_OBJECT_TYPE_LAG_MEMBER][SAI_LAG_MEMBER_ATTR_LAG_ID] = SAI_SERIALIZATION_TYPE_OBJECT_ID;
     map[SAI_OBJECT_TYPE_LAG_MEMBER][SAI_LAG_MEMBER_ATTR_PORT_ID] = SAI_SERIALIZATION_TYPE_OBJECT_ID;
 
+
+    map[SAI_OBJECT_TYPE_POLICER][SAI_POLICER_ATTR_METER_TYPE] = SAI_SERIALIZATION_TYPE_UINT32;
+    map[SAI_OBJECT_TYPE_POLICER][SAI_POLICER_ATTR_MODE] = SAI_SERIALIZATION_TYPE_UINT32;
+    map[SAI_OBJECT_TYPE_POLICER][SAI_POLICER_ATTR_COLOR_SOURCE] = SAI_SERIALIZATION_TYPE_UINT32;
+    map[SAI_OBJECT_TYPE_POLICER][SAI_POLICER_ATTR_CBS] = SAI_SERIALIZATION_TYPE_UINT64;
+    map[SAI_OBJECT_TYPE_POLICER][SAI_POLICER_ATTR_CIR] = SAI_SERIALIZATION_TYPE_UINT64;
+    map[SAI_OBJECT_TYPE_POLICER][SAI_POLICER_ATTR_PBS] = SAI_SERIALIZATION_TYPE_UINT64;
+    map[SAI_OBJECT_TYPE_POLICER][SAI_POLICER_ATTR_PIR] = SAI_SERIALIZATION_TYPE_UINT64;
+    map[SAI_OBJECT_TYPE_POLICER][SAI_POLICER_ATTR_GREEN_PACKET_ACTION] = SAI_SERIALIZATION_TYPE_UINT32;
+    map[SAI_OBJECT_TYPE_POLICER][SAI_POLICER_ATTR_YELLOW_PACKET_ACTION] = SAI_SERIALIZATION_TYPE_UINT32;
+    map[SAI_OBJECT_TYPE_POLICER][SAI_POLICER_ATTR_RED_PACKET_ACTION] = SAI_SERIALIZATION_TYPE_UINT32;
+    map[SAI_OBJECT_TYPE_POLICER][SAI_POLICER_ATTR_ENABLE_COUNTER_LIST] = SAI_SERIALIZATION_TYPE_INT32_LIST;
+
+    map[SAI_OBJECT_TYPE_TRAP_GROUP][SAI_HOSTIF_TRAP_GROUP_ATTR_ADMIN_STATE] = SAI_SERIALIZATION_TYPE_BOOL;
+    //map[SAI_OBJECT_TYPE_TRAP_GROUP][SAI_HOSTIF_TRAP_GROUP_ATTR_QUEUE] = SAI_SERIALIZATION_TYPE_UINT32;
```

Comment:
I've tested code with SAI_HOSTIF_TRAP_GROUP_ATTR_PRIO which should be removed soon and replaced with SAI_HOSTIF_TRAP_GROUP_ATTR_QUEUE new attribute.
It's not present in the current saitypes.h installed on the switch.
Forgot to delete the commented line.